docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -50,34 +65,28 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
     is a shorthand for a `UNION ALL` where each leg of the `UNION ALL`
     operator performs aggregation of subset of the columns specified in the `GROUPING SETS` clause.
```

Comment:
The example given here is too simple: `GROUP BY GROUPING SETS (warehouse, product)`.

Can we also explain `GROUP BY GROUPING SETS ((warehouse, product), (product), ())`?

Reply:
Done

Reply:
Should we explain more about `group by ()`, it mean's aggregate all result.

Reply:
We can replace `GROUP BY ()` with `global aggregate`